Makefile

```diff
@@ -14,7 +12,8 @@ KIND_CLUSTER ?= kind
 CI_MODE_ENABLED := ""
 NO_KIND_CLEANUP := ""
 
-IMG ?= ghcr.io/kube-logging/telemetry-controller:0.0.11
+# Image URL to use all building/pushing image targets
+IMG ?= ghcr.io/kube-logging/telemetry-controller:dev
```

Comment:
This variable is only used for local and CI E2E test image building.
For deploying the latest operator release, we should recommend using the OCI Helm chart release.

There’s no point to tag this with the latest release; instead, we should use `dev`, which would serve as a universal tag for local image building.

In the `logging-operator`, we use:

```
# Image name to use for building/pushing image targets
IMG ?= controller:local
```

To keep consistency, we should adopt the same pattern and use `controller:local`.

Reply:
And with that, please check that we dont have any more hardcoded version of the operator left in the code, because that makes creating release tags cumbersome

Reply:
I've found a couple of issues with the e2e setup of this project and opened a ticket about it, I've incorporated your idea in that one, IMO it's somewhat the same scope.